apps/scan/frontend/src/screens/election_manager_screen.tsx

```diff
@@ -151,10 +151,7 @@ export function ElectionManagerScreen({
   const ballotMode = (
     <P>
       <SegmentedButton
-        disabled={
-          setTestModeMutation.isLoading ||
-          (isCvrSyncRequired && scannerStatus.ballotsCounted > 0)
-        }
+        disabled={setTestModeMutation.isLoading || isCvrSyncRequired}
```

Comment:
`isCvrSyncRequired` can only be true if `ballotsCounted` is greater than 0:

https://github.com/votingworks/vxsuite/blob/d5c32154efabec320cc303c50f8648097c01b9e2/libs/backend/src/cast_vote_records/export.ts#L847-L850